xmtp_mls/src/hpke.rs

```diff
@@ -49,7 +49,7 @@ pub fn decrypt_welcome(
         .read::<HpkePrivateKey>(hpke_public_key)
         .ok_or(HpkeError::KeyNotFound)?;
 
-    let ciphertext = HpkeCiphertext::tls_deserialize_exact(ciphertext)?;
+    let ciphertext = HpkeCiphertext::tls_deserialize_exact(ciphertext).unwrap();
```

Comment:
We're trying to keep this codebase unwrap free. If the `?` doesn't work anymore it's probably because you need a new error variant that supports the error type from `tls_deserialize_exact`